src/actor.js

```diff
@@ -19,6 +22,75 @@ const tryParseDate = (str) => {
     return unix > 0 ? new Date(unix) : undefined;
 };
 
+/**
+ * Event emitter providing access to events from Actor infrastructure. Event emitter is initiated by Apify.main().
+ * If you don't use Apify.main() then you must call `await Apify.initializeEvents()` yourself.
+ *
+ * Example usage:
+ *
+ * ```javascript
+ * import { ACTOR_EVENTS } from 'apify/constants';
+ *
+ * Apify.main(async () => {
+ *   &nbsp;
+ *   Apify.events.on(ACTOR_EVENTS.CPU_INFO, (data) => {
```

Comment:
Looking into apify-shared-js the event name is "CPU_INFO". I think there's a convention to use lower case names, e.g. "cpuinfo"

Reply:
Yeah. I didnt realize this. But looking now in the Node documentation they are talking about camel case names:

"All objects that emit events are instances of the EventEmitter class. These objects expose an eventEmitter.on() function that allows one or more functions to be attached to named events emitted by the object. Typically, event names are camel-cased strings but any valid JavaScript property key can be used."
https://nodejs.org/api/events.html

So maybe camel-cased "cpuInfo" ?

Reply:
sure thing